src/ui-kit/Button/Button.styles.js

```diff
@@ -78,41 +78,63 @@ const buttonTypeProp = ({ theme, type }) => {
           border: ${darken(theme.colors.text.secondary, 0.06)};
         }
       `;
-    case 'link':
+    case "cta":
```

Comment:
We should just use the primary button type instead of adding a specific one for CTAs.

Reply:
Removed the specific cases